projects/pastanaga-angular/src/lib/modal/modal-dialog/modal-dialog.component.scss

```diff
@@ -9,7 +9,11 @@ $header-height-medium: $padding-modal + $dimension-image + $spacing-dialog-secti
     .pa-modal-header {
         @include fixed-header;
         padding: $padding-modal $padding-modal $spacing-dialog-sections;
-        text-align: center;
+        text-align: left;
+
+        &-centered {
```

Comment:
I would recommend not to use this kind of shortcut: it makes it hard to find CSS classes using the IDE navigation and search tools.
Better to write the whole class name

Reply:
done